packages/core/crates/core-types/src/channels.rs

```diff
@@ -184,13 +188,12 @@ pub fn generate_channel_id(source: &Address, destination: &Address) -> Hash {
 
 /// Contains the overall description of a ticket with a signature
 #[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
-#[cfg_attr(feature = "wasm", wasm_bindgen::prelude::wasm_bindgen(getter_with_clone))]
 pub struct Ticket {
     pub channel_id: Hash,
     pub amount: Balance,
     pub index: u64,
     pub index_offset: u32,
-    pub win_prob: f64,
+    pub encoded_win_prob: [u8; 7],
```

Comment:
Wouldn't it deserve an alias?

Reply:
the "to float" conversion is lossy, this is why there's a method `win_prob()`

Reply:
I mean type alias for the 7 byte array, calling it WinProbEncoding or smth

Reply:
makes sense